config/prismjs/usage.js

```diff
@@ -9,5 +9,13 @@ Prism.languages.usage = {
   },
   usage: {
     pattern: /(^|\n)\s*(usage|positional arguments|optional arguments)/
+  },
+  args: {
+    pattern: /(?<=\[)(?:[^\]\[]+|\[(?:[^\]\[]+|\[[^\]\[]*\])*\])*(?=\])/,
+    inside: {
+      arg: {
+        pattern: /\-{1,2}[a-zA-Z- <>\]\[.]*/
```

Comment:
Some of our option arguments are more complex than this. ([Example in `/exps/show`](https://dvc-org-interactive-syn-s47nhu.herokuapp.com/doc/command-reference/exp/show)) We might want to consider expanding this regex to include `{}`, `/`, `,`, and `_`.

![image](https://user-images.githubusercontent.com/43496356/148588195-2e8da35b-c9f3-4871-885c-b5ad85f05fba.png)

Reply:
Thanks for pointing this out. I have simply generalized it to include all.